workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardTable.tsx

```diff
@@ -3,8 +3,10 @@ import ClayTable from '@clayui/table';
 import swapVert from '../../assets/icons/swap-vert.svg';
 
 import './DashboardTable.scss';
+
+import React, {ReactNode} from 'react';
+
 import {DashboardEmptyTable} from './DashboardEmptyTable';
```

Comment:
hey @naraandrade, I would like to know if you can see what is breaking in the source format.
Before submitting the PR, I tried to reorder `import {DashboardEmptyTable} from './DashboardEmptyTable';` to be below `import './DashboardTable.scss';`. But after running the liferay format it reordered this way.

Reply:
Resolved by:

https://liferay.slack.com/archives/C19PWQ74J/p1678994440211439?thread_ts=1678991765.934409&cid=C19PWQ74J